app/src/main/java/at/tugraz/ist/swe/Circle.java

```diff
@@ -0,0 +1,37 @@
+package at.tugraz.ist.swe;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Point;
+import android.view.MotionEvent;
+
+public class Circle extends PaintingTool {
+
+    Point position;
```

Comment:
Would make this member private.

```suggestion
    private Point position;
```